src/app/app.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet],
```

Comment:
issue

          styleUrls: ['./app.component.scss']

          Explantion: The Angular property for specifying styles should be 'styleUrls' and not 'styleUrl'.